### ISSUE TITLE
ENOENT for piped output

### ISSUE THREAD

**Issue by Queuecumber** [clutchski/coffeelint#263]
I originally posted this issue to nodejs [here](https://github.com/joyent/node/issues/7488) and was asked to repost it here instead. 

When running the following command 

```
coffeelint -f <( echo $COFFEELINT_SETTINGS ) .
```

The following error is reported:

```
fs.js:679
return binding.lstat(pathModule._makeLong(path));
             ^
Error: ENOENT, no such file or directory '/proc/21512/fd/pipe:[140746056]'
  at Object.fs.lstatSync (fs.js:679:18)
  at Object.realpathSync (fs.js:1267:21)
  at read (/usr/lib/node_modules/coffeelint/lib/commandline.js:35:19)
  at Object.<anonymous> (/usr/lib/node_modules/coffeelint/lib/commandline.js:447:29)
  at Object.<anonymous> (/usr/lib/node_modules/coffeelint/lib/commandline.js:476:4)
  at Module._compile (module.js:456:26)
  at Object.Module._extensions..js (module.js:474:10)
  at Module.load (module.js:356:32)
  at Function.Module._load (module.js:312:12)
  at Module.require (module.js:364:17)
  at require (module.js:380:17)
  at Object.<anonymous> (/usr/lib/node_modules/coffeelint/bin/coffeelint:17:5)
  at Module._compile (module.js:456:26)
  at Object.Module._extensions..js (module.js:474:10)
  at Module.load (module.js:356:32)
  at Function.Module._load (module.js:312:12)
  at Function.Module.runMain (module.js:497:10)
  at startup (node.js:119:16)
  at node.js:902:3
```

According to them, readLink is being called on the input file which is causing a problem when the input is piped the way I'm using. That being said I am not convinced this is a coffeelint issue. This issue would effect anyone using bash.

One more note, this was intended as a workaround for [this closed issue](https://github.com/clutchski/coffeelint/issues/216), it should really be directly supported as it is incredibly useful to have, but if not this should at least be fixed by whoever's module is at fault.
